### PR TITLE
feat: Code format

### PR DESCRIPTION
**Pull request by fengshunli**
Signed-off-by: fengshunli <1171313930@qq.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: cmd/util/applicationset_test.go [hunk 1]
```diff
@@ -31,10 +31,10 @@ spec:
 `
 
 func TestReadAppSet(t *testing.T) {
-	appsets := []*argoprojiov1alpha1.ApplicationSet{}
-	err := readAppset([]byte(appSet), &appsets)
+	var appset []*argoprojiov1alpha1.ApplicationSet
```
Comment (crenshaw-dev) [inline cmd/util/applicationset_test.go:34]: I think `appSets` makes more sense. This is a slice of appsets.

File: cmd/argocd/commands/bcrypt_test.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ func TestGeneratePassword(t *testing.T) {
 	bcryptCmd := NewBcryptCmd()
 	bcryptCmd.SetArgs([]string{"--password", "abc"})
 	output := new(bytes.Buffer)
-	bcryptCmd.SetOutput(output)
+	bcryptCmd.SetOut(output)
```
[the remark this answers is not in the document]
Comment (fengshunli) [reply cmd/argocd/commands/bcrypt_test.go:15]: // Deprecated: Use SetOut and/or SetErr instead
image

File: cmd/util/applicationset_test.go (hunk 1 above)
Comment (fengshunli) [reply cmd/util/applicationset_test.go:34]: Empty slice declaration using a literal

Comment (crenshaw-dev) [reply cmd/util/applicationset_test.go:34]: Yep, the change of the declaration is fine. I think changing the variable name was incorrect though.

Comment (fengshunli) [reply cmd/util/applicationset_test.go]: @crenshaw-dev done

**Review by crenshaw-dev — APPROVED** [review]

**Pull request merged**